src/gateways/PrismaGouvernanceRepository.test.ts

```diff
@@ -81,12 +82,71 @@ describe('gouvernance repository', () => {
     await prisma.noteDeContexteRecord.create({
       data: noteDeContexteRecordFactory({ gouvernanceId, id: 1 }),
     })
-    const repository = new PrismaGouvernanceRepository(prisma.gouvernanceRecord)
+    const repository = new PrismaGouvernanceRepository(prisma.gouvernanceRecord, prisma.noteDeContexteRecord)
 
     // WHEN
     const gouvernanceTrouvee = await repository.find(new GouvernanceUid(String(gouvernanceId)))
 
     // THEN
     expect(gouvernanceTrouvee?.state).toStrictEqual(gouvernanceFactory({ uid: '1' }).state)
   })
+
+  it('ajouter une note de contexte à une gouvernance', async () => {
+    // GIVEN
+    const gouvernanceId = 1
+    await prisma.regionRecord.create({
+      data: regionRecordFactory({ code: '11' }),
+    })
+    await prisma.departementRecord.create({
+      data: departementRecordFactory({ code: '75' }),
+    })
+    await prisma.utilisateurRecord.create({
+      data: utilisateurRecordFactory({ id: 1, ssoId: 'userFooId' }),
+    })
+
+    await prisma.gouvernanceRecord.create({
+      data: gouvernanceRecordFactory({ departementCode: '75', id: gouvernanceId }),
+    })
+
+    const repository = new PrismaGouvernanceRepository(prisma.gouvernanceRecord, prisma.noteDeContexteRecord)
+    const gouvernanceMiseAJourAvecNoteDeContexte = gouvernanceFactory({
+      noteDeContexte: {
+        contenu: 'contenu',
+        dateDeModification: new Date(),
+        uidEditeur: new UtilisateurUid({
+          email: 'martin.tartempion@example.net',
+          value: 'userFooId',
+        }),
+      },
+      uid: String(gouvernanceId),
+    })
+
+    // WHEN
+    await repository.update(gouvernanceMiseAJourAvecNoteDeContexte)
+
+    // THEN
+    const gouvernanceMiseAJour = await repository.find(new GouvernanceUid(String(gouvernanceId)))
+    expect(gouvernanceMiseAJour?.state).toStrictEqual(gouvernanceMiseAJourAvecNoteDeContexte.state)
```

Comment:
Tu n'es pas censé savoir que `find` existe et de manière générale, il ne faut pas utiliser du code de production dans un test.
Du coup, ça donnerait ça :
```suggestion
    const gouvernanceMiseAJour = await prisma.gouvernanceRecord.find(new GouvernanceUid(String(gouvernanceId)))
    expect(gouvernanceMiseAJour?.state).toStrictEqual(gouvernanceRecordFactory(...))
```
Tu peux regarder dans le repo de comité pour avoir des exemples.